pensieve/cli.py

```diff
@@ -30,7 +31,15 @@ def inclusive_date_range(start_date, end_date):
 
 
 def default_spec_for_experiment(experiment: experimenter.Experiment) -> AnalysisSpec:
-    return AnalysisSpec.from_dict(toml.load(DEFAULT_METRICS_CONFIG))
+    default_metrics = AnalysisSpec.from_dict(toml.load(DEFAULT_METRICS_CONFIG))
+
+    if experiment.type == "message":
+        # CFR experiment
+        cfr_metrics = AnalysisSpec.from_dict(toml.load(CFR_METRICS_CONFIG))
+        default_metrics.merge(cfr_metrics)
```

Comment:
I'm guessing we still want to calculate the default metrics for CFR experiments and add a few specific ones on top of that?

Reply:
That sounds right to me.